maquetado/detail.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+
+<html lang="en">
+<head>
```

Comment:
Add two spaces to all lines that appear inside `html` tag

Reply:
Done.